content/en/docs/How-to guides/clone-repository.md

```diff
@@ -0,0 +1,476 @@
+    [fetch-source : clone] {"level":"info","ts":1652300245.5349698,"caller":"git/git.go:208","msg":"Successfully initialized and updated submodules in path /workspace/output/"}
+    [fetch-source : clone] + cd /workspace/output/
+    [fetch-source : clone] + git rev-parse HEAD
+    [fetch-source : clone] + RESULT_SHA=4930334b17edeaa737e2e6d0c7f7139b0afb1896
+    [fetch-source : clone] + EXIT_CODE=0
+    [fetch-source : clone] + '[' 0 '!=' 0 ]
+    [fetch-source : clone] + printf '%s' 4930334b17edeaa737e2e6d0c7f7139b0afb1896
+    [fetch-source : clone] + printf '%s' https://github.com/tektoncd/website
+
+    [show-readme : read] # TektonCD Website
+    [show-readme : read] 
+    [show-readme : read] This repo contains the code behind [the Tekton org's](https://github.com/tektoncd)
+    [show-readme : read] website at [tekton.dev](https://tekton.dev).
+    [show-readme : read] 
+    [show-readme : read] For more information on the Tekton Project, see
+    [show-readme : read] [the community repo](https://github.com/tektoncd/community).
+    [show-readme : read] 
+    [show-readme : read] For more information on contributing to the website see:
+    [show-readme : read] 
+    [show-readme : read] * [CONTRIBUTING.md](CONTRIBUTING.md)
+    [show-readme : read] * [DEVELOPMENT.md](DEVELOPMENT.md)
+    ```
+
+## Full code samples
+
+This is the full code for the Pipeline:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: clone-read
+spec:
+  description: | 
+    This pipeline clones a git repo, then echoes the README file to the stout.
+  params:
+  - name: repo-url
+    type: string
+    description: The git repo URL to clone from.
+  workspaces:
+  - name: shared-data
+    description: | 
+      This workspace contains the cloned repo files, so they can be read by the
+      next task.
+  - name: git-credentials
+    description: My ssh credentials
+  tasks:
+  - name: fetch-source
+    taskRef:
+      name: git-clone
+    workspaces:
+    - name: output
+      workspace: shared-data
+    - name: ssh-directory
+      workspace: git-credentials
+    params:
+    - name: url
+      value: $(params.repo-url)
+  - name: show-readme
+    runAfter: ["fetch-source"]
+    taskRef:
+      name: show-readme
+    workspaces:
+    - name: source
+      workspace: shared-data
+```
+
+And for the PipelineRun:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: clone-read-run
+spec:
+  pipelineRef:
+    name: clone-read
+  workspaces:
+  - name: shared-data
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+  - name: git-credentials
+    secret:
+      secretName: git-credentials
+  params:
+  - name: repo-url
+    value: git@github.com:tektoncd/website.git
+```
+
+## Further reading
+
+- [Build and push a Docker container image with Kaniko and Tekton
```

Comment:
This doesn't exist yet as far as I can see

Reply:
It's currently work in progress. I changed the link to the Pipeline docs instead and will update it when that how-to is published.